### PR TITLE
.github/: Create PULL_REQUEST_TEMPLATE.md

### LINKED ISSUE #41: Create a pull request template
## Description:
as a developer
when I make a pr
I want the body of the PR to have a PR template for me to complete

## Tasks:
- [ ] Add a PR template

## Resources: 
- [ ] [a list of pr template examples from GH](https://github.com/stevemao/github-issue-templates)

## Additional remarks:
Probably a good initial setup step to allow people to create issues

### PR DESCRIPTION
**Pull request by Philip-Clark**
## Because

PR templates enable uniform and efficient PR's.

## This PR

- Create PULL_REQUEST_TEMPLATE.md
- Adapt The Odin Project PR template to suit Odin Pantry's needs

## Issue

Closes #41 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Menu.js: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!-- Thank you for taking the time to contribute to Odin-pantry. Please complete this entire template. -->
+
+## Because
```
Comment (MarchandMD) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Hmmm, this is perhaps confusing to someone not used to PRs and using a PR template. 

What do you think about giving this section a title such as "PR reason", or "What this PR introduces" or some other title...

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<!-- Summarize the purpose or reasons for this PR, e.g. what problem it solves or what benefit it provides. -->
+
+
+## This PR
```
Comment (MarchandMD) [inline .github/PULL_REQUEST_TEMPLATE.md:7]: same suggestion as before; maybe something a little more obvious such as "What changes in this PR", etc

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+<!-- A bullet point list of one or more items describing the specific changes. -->
+- e.g. Add PR templates
+
+
+## Issue
+<!--
```
Comment (MarchandMD) [inline .github/PULL_REQUEST_TEMPLATE.md:13]: I've seen a team go so far as to include the `#(issue)` on this section, and give instructions to the submitter to replace `(issue)` so they can see the magic of Github make the connection

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+If this PR does not close but is related to another issue or PR, you can link it as above without the 'Closes' keyword, e.g. 'Related to #2013'.
+
+-->
+Closes #XXXXX
+
+## Additional Information
+<!-- Any other information about this PR, such as a link to a Discord discussion. -->
+
+
+## Pull Request Requirements
```
Comment (MarchandMD) [inline .github/PULL_REQUEST_TEMPLATE.md:25]: Love this section! Very helpful!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+## Issue
+<!--
+If this PR closes an open issue in this repo, replace the XXXXX below with the issue number, e.g. Closes #2013.
+
+If this PR does not close but is related to another issue or PR, you can link it as above without the 'Closes' keyword, e.g. 'Related to #2013'.
+
+-->
+Closes #XXXXX
+
+## Additional Information
```
Comment (MarchandMD) [inline .github/PULL_REQUEST_TEMPLATE.md:21]: perhaps adding "optional" text, so people know things are optional?

**Review by MarchandMD — COMMENTED** [review]
just a couple of simple thoughts; let me know what you think

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Philip-Clark) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: Sounds reasonable. I'll update that.
This is an adaptation of TOP's PR template, so I didn't change a lot of the "legalistic" wording. I'll do my best to make it more beginner friendly given the purpose of this project.

Comment (Philip-Clark) [reply .github/PULL_REQUEST_TEMPLATE.md]: Updated a major chunk of the wording and added examples in every section to help explain things. Let me know what you think.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Philip-Clark) [reply .github/PULL_REQUEST_TEMPLATE.md]: Changed to: Changes in this PR

**Review by MarchandMD — APPROVED** [review]
I think this is great! If other people feel differently/strongly about any of it, we can always re-visit

**Pull request merged**